src/utils/cli-action.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+import cli from 'cli-ux';
+
+export async function action(output: string, showSpinner: boolean, promise: Promise<any>) {
```

Comment:
1. Currently this function returns `Promise<void>`. You might want to consider returning the result of the promise.
2. Maybe generics could come in handy here?
3. Maybe showSpinner could have a default value? (depending on which is more common).

```suggestion
export async function action<T>(output: string, promise: Promise<T>, showSpinner = false): Promise<T> {
  if (showSpinner) {
    cli.action.start(output);
  } else {
    cli.log(`${output}...`);
  }
  const promiseResult = await promise;
  if (showSpinner) {
    cli.action.stop();
  }
  
  return promiseResult;
}
```